src/App.tsx

```diff
@@ -1,24 +1,10 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import UserName from "./components/UserName"
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+     <UserName minSymbol={3} maxSymbol={256}/>
```

Comment:
it is better to use const for storing minSymbol and maxSymbol:

const USERNAME = {
MIN: 3,
MAX: 256,
}

UserName minSymbol={USERNAME.MIN} maxSymbol={USERNAME.MAX}